dsls/sql/src/main/java/org/apache/beam/dsls/sql/BeamSql.java

```diff
@@ -188,8 +226,12 @@ private void validateQuery() {
 
     @Override
     public PCollection<BeamSqlRow> expand(PCollection<BeamSqlRow> input) {
+      validateQuery();
       return PCollectionTuple.of(new TupleTag<BeamSqlRow>(PCOLLECTION_TABLE_NAME), input)
-          .apply(new QueryTransform(sqlQuery, sqlEnv));
+          .apply(new AutoValue_BeamSql_QueryTransform.Builder()
```

Comment:
Ping on this comment, since it wasn't addressed or answered. Can't this be simplified?